examples/twittermap/web/app/views/twittermap/main.scala.html

```diff
@@ -69,6 +70,14 @@
   <script src='@routes.Assets.versioned("javascripts/cache/services.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/mapresultcache/services.js")'></script>
   <script src='@routes.Assets.versioned("javascripts/map/controllers.js")'></script>
+  <script src='@routes.Assets.versioned("javascripts/lib/QuadTree.js")'></script>
```

Comment:
Please put a comment about this external javascript. What does it do and where it is used.

Reply:
Sorry, do you mean use a comment inside this main.scala.html code file? But I see there is not any comment before any lines, don't you think it's a little not consistent with the beauty of this file?

Reply:
Got the point. You may create a wiki page about the pointmap. :-)

Reply:
OK. I will do that later.

Reply:
done